src/features/fix-all.ts

```diff
@@ -3,14 +3,15 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import tsp, { CommandTypes } from 'typescript/lib/protocol';
-import * as lsp from 'vscode-languageserver/node';
-import { LspDocuments } from '../document';
-import { toFileRangeRequestArgs, toTextDocumentEdit } from '../protocol-translation';
-import { TspClient } from '../tsp-client';
-import * as errorCodes from '../utils/errorCodes';
-import * as fixNames from '../utils/fixNames';
-import { CodeActionKind } from '../utils/types';
+import type tsp from 'typescript/lib/protocol.d.js';
```

Comment:
Why do you need to import the d.ts? it seems uncommon

Reply:
It would work if I'd use the `.js` extension but in this case the `protocol.js` file doesn't exist physically, only `protocol.d.ts` so this seems a bit more explicit in describing what we are doing.

Reply:
LGTM then